Taskfile.yml

```diff
@@ -143,7 +144,7 @@ tasks:
     run: once
     deps: [gen-version]
     cmds:
-      - go test -c -o=bin/ferretdb{{exeExt}} {{.RACE_FLAG}} -tags=ferretdb_testcover,{{.BUILD_TAGS}} -coverpkg=./... ./cmd/ferretdb
+      - go test -c -o=bin/ferretdb{{exeExt}} {{.RACE_FLAG}} -tags=ferretdb_debug,{{.BUILD_TAGS}} -coverpkg=./... ./cmd/ferretdb
```

Comment:
That should be replaced with `go build -cover`

Reply:
`go test -c` plus main_test.go file is what allowed us to gather coverage of the running binary: https://www.elastic.co/blog/code-coverage-for-your-golang-system-tests But Go 1.20 added `go build -cover` that makes it simpler, so let’s make it simpler